src/test/java/christmas/domain/date/VisitDayTest.java

```diff
@@ -0,0 +1,59 @@
+package christmas.domain.date;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+
+import christmas.domain.date.domain.VisitDate;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class VisitDayTest {
+
+    @DisplayName("입력한 날짜가 1 ~ 31 범위면 에러를 반환하지 않는다.")
+    @ParameterizedTest
+    @ValueSource(ints = {1, 25, 31})
+    void createSuccessTest(int day) {
+        assertThatCode(() -> new VisitDate(day))
+                .doesNotThrowAnyException();
+    }
+
+    @DisplayName("입력한 날짜가 1 ~ 31 범위가 아니면 에러를 반환한다.")
+    @ParameterizedTest
+    @ValueSource(ints = {-1, 0, 32})
+    void createFailTest(int day) {
+        assertThatCode(() -> new VisitDate(day))
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessage("[ERROR] 유효하지 않은 날짜입니다. 다시 입력해 주세요.");
+    }
+
+    @ParameterizedTest
+    @MethodSource("dayOfWeekTestArguments")
+    void dayOfWeekTest(int day, boolean expectedIsWeekEnd) {
+        VisitDate visitDate = new VisitDate(day);
+        assertThat(visitDate.isWeekEnd()).isEqualTo(expectedIsWeekEnd);
+    }
```

Comment:
검증부와 준비하는 부분의 개행을 하면 가독성이 좋을 것 같아요
```suggestion
    @ParameterizedTest
    @MethodSource("dayOfWeekTestArguments")
    void dayOfWeekTest(int day, boolean expectedIsWeekEnd) {
        VisitDate visitDate = new VisitDate(day);
        
        assertThat(visitDate.isWeekEnd()).isEqualTo(expectedIsWeekEnd);
    }
```

Reply:
테스트코드 가독성도 신경쓰도록 노력하겠습니다. 
바쁘신 기간에 리뷰 해주셔서 너무 감사드립니다 박스터님!
새해 복 많이 받으세요!!